tools/ot/src/main/java/dk/alexandra/fresco/tools/ot/base/Ot.java

```diff
@@ -0,0 +1,29 @@
+package dk.alexandra.fresco.tools.ot.base;
+
+import dk.alexandra.fresco.framework.util.StrictBitVector;
+
+/**
+ * Oblivious Transfer interface.
+ */
+public interface Ot {
+
+  /**
+   * Send two possible messages.
```

Comment:
Complete the javadoc - ... for the reciepient to choose from".
Ot is not about sending two possible messages.

Reply:
Added